internal/ingress/controller/template/template_test.go

```
@@ -178,6 +180,11 @@ proxy_pass http://upstream_balancer;`,
 	}
 )
 
+func getTestDataDir() string {
+	pwd, _ := os.Getwd()
```

Please do not ignore errors